test/unit/DetectChanges/BCBreak/FunctionBased/ParameterDefaultValueChangedTest.php

```diff
@@ -71,7 +74,12 @@ function positionOfOptionalParameterChanged($a = 2, $b, $c = 1) {}
    class C {
        static function changed1($a = 1) {}
        function changed2($a = 1) {}
+       function notChangedNewInitializer($a = new stdClass()) {}
+       function notChangedNewEnum($a = D::A) {}
```

Comment:
We need to test changes too

Reply:
Added two testcase which both dont report changes as expected